scripts/data_prep/convert_dataset_hf.py

```diff
@@ -316,7 +324,11 @@ def main(args: Namespace) -> None:
 
     if args.concat_tokens is not None:
         mode = ConcatMode.CONCAT_TOKENS
-        tokenizer = AutoTokenizer.from_pretrained(args.tokenizer)
+        if args.tokenizer == 'tiktoken':
```

Comment:
can you call build_tokenizer here instead of inlining this logic?

Reply:
yes, yes I can 🤦